src/reflow/mod.rs

```diff
@@ -59,11 +76,25 @@ fn reflow<'s>(origin: ContentOrigin, chunk: &'s CheckableChunk) -> Vec<Suggestio
                 start: paragraph,
                 end,
             };
+
+            let spans = chunk.find_spans_inclusive(range.clone());
+            let span = Span {
```

Comment:
we can't really guarantee it's none empty, can we? If so, let's use `expect("Does not X. qed")`